### ISSUE TITLE
How to Use the files?

### ISSUE THREAD

**Issue by ludusrusso** [trodi/blockly-d.ts#3]
Hey guys,
sorry I'm a real newbbo in typescript.
I don't understand how can I include the generated file in a Typescript folder.. I'm developing an Angular application and I don't understand where to put this file to use blockly!

**Comment by trodi** [comment]
* Remember: You don't need the definition file to *use* Blockly in your typescript project. This will only give TSC type information so if can provide type checking for Blockly (aka, your project would run, you would just have TSC errors). You'll still need to include the Blockly source code. Ideally, you'd prefer to match the version of the definition file to the version of Blockly you are using.
* To use this definition file, you need to add it to your project and make sure TSC knows about. The file from this repo you need is dist/blockly-core.d.ts.
  * There are a number of ways to tell TSC about files depending on your project setup. The official [handbook](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html) explains how files are added and specifically definition files (d.ts files).
  * Normally you would follow the directions [here](https://www.typescriptlang.org/docs/handbook/declaration-files/consumption.html) to simply grab them from `npm`, however these definitions aren't currently on `npm`.

Hope this helps. I'm interested to know what parts of the readme instructions were confusing so I can improve them. Thanks.

**Comment by ludusrusso** [comment]
@trodi no it's just me since I'm very new in web developing :D
Anyway, I'm trying to integrate blockly in an Angular4.0 application, but it seems that there are some strange behavior when loading from ng. At the beginning I was thinking it was due to the definitions, but it seems not. 

If I load Blockly I get an error such as `Blockly.inject` is not a function.

At the end, I've found a workaround by simply include blockly from `<script>` tag in html!
I'll investigate the problem anyway!

Thanks for the help!
